packages/rocketchat-ui-flextab/client/flexTabBar.js

```diff
@@ -188,15 +188,30 @@ Template.RoomsActionTab.helpers({
 		if (Template.instance().small.get()) {
 			return [];
 		}
+		let removedButton = false;
```

Comment:
you dont need this variable

Reply:
Removing this variable creates a potential issue. When the `add users` button isn't displayed, then `files list` gets duplicated. It stays in it's original `more` section and takes the place of the `add users` button